plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGenerator.vue

```diff
@@ -0,0 +1,618 @@
+           target="_blank" rel="noopener">Google Tag Manager</a>
+      </p>
+
+      <Field
+        uicontrol="site"
+        name="js-tracker-website"
+        class="jsTrackingCodeWebsite"
+        v-model="site"
+        :introduction="translate('General_Website')"
+      />
+
+      <div id="javascript-output-section">
+        <div class="valign-wrapper trackingHelpHeader matchWidth">
+          <div>
+            <h3>{{ translate('General_JsTrackingTag') }}</h3>
+
+            <p>
+              {{ translate('CoreAdminHome_JSTracking_CodeNoteBeforeClosingHead', "&lt;/head&gt;") }}
+            </p>
+          </div>
+
+          <button class="btn" id="emailJsBtn" @click="sendEmail()">
+            {{ translate('SitesManager_EmailInstructionsButton') }}
+          </button>
+        </div>
+        <div id="javascript-text">
+          <pre v-select-on-focus="{}" class="codeblock" v-text="trackingCode" ref="trackingCode"/>
+        </div>
+      </div>
+
+      <div id="optional-js-tracking-options">
+        <!-- track across all subdomains -->
+        <div id="jsTrackAllSubdomainsInlineHelp" class="inline-help-node">
+          <span v-html="mergeSubdomainsDesc"></span>
+          <span v-html="learnMoreText"></span>
+        </div>
+
+        <Field
+          uicontrol="checkbox"
+          name="javascript-tracking-all-subdomains"
+          :model-value="trackAllSubdomains"
+          @update:model-value="trackAllSubdomains = $event; updateTrackingCode()"
+          :disabled="isLoading"
+          :introduction="translate('General_Options')"
+          :title="`${translate(
+            'CoreAdminHome_JSTracking_MergeSubdomains',
+          )} ${site.name}`"
+          inline-help="#jsTrackAllSubdomainsInlineHelp"
+        />
+      </div>
+
+      <!-- group page titles by site domain -->
+      <div id="jsTrackGroupByDomainInlineHelp" class="inline-help-node">
+        {{ translate('CoreAdminHome_JSTracking_GroupPageTitlesByDomainDesc1', currentSiteHost) }}
+      </div>
+
+      <Field
+        uicontrol="checkbox"
+        name="javascript-tracking-group-by-domain"
+        :model-value="groupByDomain"
+        @update:model-value="groupByDomain = $event; updateTrackingCode()"
+        :disabled="isLoading"
+        :title="translate('CoreAdminHome_JSTracking_GroupPageTitlesByDomain')"
+        inline-help="#jsTrackGroupByDomainInlineHelp"
+      />
+
+      <!-- track across all site aliases -->
+      <div id="jsTrackAllAliasesInlineHelp" class="inline-help-node">
+        {{ translate('CoreAdminHome_JSTracking_MergeAliasesDesc', currentSiteAlias) }}
+      </div>
+
+      <Field
+        uicontrol="checkbox"
+        name="javascript-tracking-all-aliases"
+        :model-value="trackAllAliases"
+        @update:model-value="trackAllAliases = $event; updateTrackingCode()"
+        :disabled="isLoading"
+        :title="`${translate('CoreAdminHome_JSTracking_MergeAliases')} ${site.name}`"
```

Comment:
`site.name` needs to be encoded in some way. As visible on the UI screenshots a html would currently not be escaped correctly. Same applies for the other usage of `site.name`